qemu/tests/block_hotplug.py

```diff
@@ -228,6 +231,8 @@ def unplug_device(vm, get_disk_cmd, device):
                     test.fail("Failed to unplug disks '%s'" % device.get_param("id"))
                 device_list.remove(device)
             controller.unplug(vm.monitor)
+            time.sleep(pause)
+            controller.verify_unplug("", vm.monitor)
```

Comment:
could you please use wait_for() to check if unplug successfully?

Reply:
Hi coli, the controller.verify_unplug("", vm.monitor) will verify if the controller has been deleted, if not be deleted will report False, why need use wait_for() to check it again?

Thanks~

Reply:
wait_for() could combine time.sleep() and verify_unplug together, and which is more reasonable.

Reply:
Thanks coli, has updated the code, could you help to review it again?
thanks a lot~